test/Interfaces/invocationFlags.C

```diff
@@ -80,5 +84,18 @@ IOpts.CompilerOpts.Remaining
 // Windows translates -nostdinc++ to -nostdinc++. Ignore that fact for the test.
 // CHECK-NEXT: {{.*}} { "progname", "-xcuda", "FileToExecuteA", "-isysroot", "APAth", {{.*}}, "-v", "FileToExecuteB" }
 
+// this flag allows to compile ptx code with the interpreter instance
+// CUDAHost and CUDADevice must not be true at the same time
+// if --cuda-device-only is set, it isn't important if -xcuda is set
```

Comment:
Maybe that's easier to read:
```suggestion
// --cuda-device-only implies -xcuda
```